FRINXio/frinx-frontend
# fix wf builder labels in edit modal

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-523` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |

https://frinxhelpdesk.atlassian.net/browse/FD-523?atlOrigin=eyJpIjoiYzE4M2Q0YmVjMDdkNGI4ZGIzYjI2NjM5OTgzZTNkZDciLCJwIjoiaiJ9

labels of workflow were not shown when editing workflow via actions menu

HOW TO TEST
Open any workflow in edit mode, select actions and "Edit workflow". All labels that belong to the workflow should be visible and able to edit
